libsupport/CMakeLists.txt

```diff
@@ -32,6 +33,18 @@ target_include_directories(katana_support PUBLIC
 
 set_common_katana_library_options(katana_support)
 
+find_package(Arrow REQUIRED)
```

Comment:
Can we remove the corresponding arrow configuration in `libtsuba`?

Reply:
I'll take the `arrow::arrow` out of libtsuba.  It still needs `arrow::parquet`.

Reply:
I think this is the minimal change, but I'm not sure about the `else `case.

Reply:
On reflection, redundant import of both arrow and parquet in both libraries might not be the end of the world either (since parquet and arrow go together).

The `else` case looks okay to me.

Reply:
Cool.  I'm happy to leave it like it is, or add back arrow and parquet in both places.

Reply:
Sigh.  It took @tylershunt to realize that I can't have libsupport depend on arrow and libtsuba depend on parquet.  Now, libsupport depends on both libtsuba and parquet.  I think that'll do it.